app/src/lib/dispatcher/validated-repository-path.ts

```diff
@@ -1,15 +1,12 @@
-import * as Path from 'path'
-
-import { getGitDir } from '../git'
+import { getTopLevelWorkingDirectory } from '../git'
 
 /**
  * Get the path to the parent of the .git directory or null if the path isn't a
  * valid repository.
  */
 export async function validatedRepositoryPath(path: string): Promise<string | null> {
   try {
-    const gitDir = await getGitDir(path)
```

Comment:
It's still being used by `isGitRepository`. We could swap that to using `getTopLevelWorkingDirectory` though and get rid of `getGitDir`. I _believe_ they are interchangeable. Want me to make that change?